From a81db81addaed3ec38de977dab640e49e84e2129 Mon Sep 17 00:00:00 2001
From: Andreas Neubacher <a_neubi@gmx.at>
Date: Fri, 6 Feb 2015 09:12:23 +0000
Subject: [PATCH] fix register address

* correct address of "Peripheral Select Register 2" -> was unable to select peripheral function of portpin

Signed-off-by: Andreas Neubacher <neubacher.andreas@gmail.com>
---
 include/arch/at91_pio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/arch/at91_pio.h b/include/arch/at91_pio.h
index 6888733..d8b324f 100644
--- a/include/arch/at91_pio.h
+++ b/include/arch/at91_pio.h
@@ -59,7 +59,7 @@
 /* 0x006c */
 #ifndef CONFIG_HAS_PIO3
 #define PIO_ASR		0x0070	/* Peripheral Select Register 1 */
-#define PIO_BSR		0x007c	/* Peripheral Select Register 2 */
+#define PIO_BSR		0x0074	/* Peripheral Select Register 2 */
 #define PIO_ABSR	0x0078	/* AB Select Status Register */
 /* 0x007c ~ 0x009c */
 #else
-- 
1.9.1

